### DISCUSSION TITLE
关于License

### DISCUSSION THREAD

**Discussion by ffha** [Meloong-Git/PCL#3974] [讨论]
@LTCatt 你是否在寻找[CC BY-NC-ND 4.0](https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode.zh-hans)？

**Comment by allMagicNB** [comment]


**Reply by ffha** [reply]
修改部分条款也行吧（大概）

**Comment by LTCatt** [comment]
一方面，CC 协议不适用于代码。
另一方面，目前的协议是仅在特定情况下禁止分发，和它也对不上。
自己修改法律文件如果改炸了，风险会很蛋疼……
